useful_resources/sample_code/ngpvan_sample_list.py

```diff
@@ -0,0 +1,73 @@
+# ### METADATA
+
+# Connectors: VAN
+# Description: Creates a new saved list from a random sample of an existing saved list in VAN
+
+# ### CONFIGURATION
+
+# Set the configuration variables below or set environmental variables of the same name and leave
+# these with empty strings.  We recommend using environmental variables if possible.
```

Comment:
Not sure we should change anything here, but worth noting that items that are likely to change often (as opposed to those that are evergreen), such as `VAN_FOLDER_ID`, `VAN_SAVED_LIST_ID`, etc. are more annoying to move in and out of env vars.

Reply:
This is a very valid point, and something I ran into when writing my own script today. What @pjsier has done here is in line with the templates I made, but I think you're right that we shouldn't be storing *all* the configuration variables as environmental variables, just the authentication ones.  If we make this change, we should do it for all scripts though. So maybe keep this as is, and we can create a separate issue to switch all the scripts + the template at once?